services/QuillLMS/app/models/concept_feedback.rb

```diff
@@ -37,6 +41,31 @@ def as_json(options=nil)
     data
   end
 
+  def queue_translation
+    return if data["description"].nil?
+    return unless translation_mappings.empty?
+
+    english = EnglishText.find_by(text: desc)
+    return create_english_text_and_return_payload unless english.present?
+
+    translation_mappings.create(english_text: english)
+    nil
+  end
+
+  def fetch_translation!
```

Comment:
Why not `fetch_translations!` plural?  

Also, one liner seems readable:
`def fetch_translations! = translated_texts.each(&:fetch_translation!)`

Reply:
for the moment it's really a one-to-one relationship, but the relationship is a has_many so there *could* be more than one. I'll change it to be plural.